kdenlive/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "2.1"
+
+services:
+  server:
+    image: linuxserver/kdenlive:421.12.3
+    restart: on-failure
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - SUBFOLDER=/
```

Comment:
I can't find the docs for this Docker image on `linuxserver.io` to understand the env. vars. Would you know where to find more info? I'm particularly interested in what the `SUBFOLDER` var. does.

Reply:
That is something that can be used with "Reverse Proxy" which I am planning to test in the coming days.

Reply:
Can you explain a bit more how `SUBFOLDER` is used with a reverse proxy?

Reply:
Yes. So when a reverse proxy is configured the `SUBFOLDER` tag specifies which of the folders can be accessed remotely i.e. limit the access to that folder and not others when accessing from outside the LAN. 

I have kept it here for now so that we can refer and update it later on. If you want we can comment that out as well, no issues. Won't be a problem in running the app.